templates/taxi/car_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Create Car</h1>
+  <form action="" method="post" novalidate>
+  {% csrf_token %}
+  {{ form.as_p }}
+    <input type="submit" value="Submit">
+  </form>
+{% endblock %}
```

Comment:
Add newline to all files.

![image](https://user-images.githubusercontent.com/104218550/203133972-4652b57a-da50-45e4-9093-476ea67ea436.png)